.github/workflows/ci.yml

```diff
@@ -58,6 +58,8 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install setuptools on Python 3.12
+        run: python3 -c 'import setuptools' || python3 -m pip install setuptools
```

Comment:
Shouldn't we add this to [install documentation](https://avocado-framework.readthedocs.io/en/latest/guides/user/chapters/installing.html#installing)?

Reply:
I've added the following:

setuptools is an Avocado dependency, but it’s no longer available by default on Python 3.12 and later. Please make sure you install it, say with pip install setuptools, before attempting to install Avocado itself.